p11-kit/meson.build

```diff
@@ -93,17 +93,23 @@ p11_module_ldflags = cc.get_supported_link_arguments([
 ])
 p11_module_symbol_def = meson.current_source_dir() / 'p11-module.def'
 
+p11v3_module_symbol_map = meson.current_source_dir() / 'p11v3-module.map'
```

Comment:
AFAIK the reasoning here was to verify backward compatibility, mostly in the testing as some of the mock pkcs11 modules still support only v2, while some support the v3. If this will not cause issues, I can change this to use only the old names here.